clients/src/main/java/org/apache/kafka/common/config/SslConfigs.java

```diff
@@ -41,8 +40,10 @@
      * @deprecated As of 1.0.0. This field will be removed in a future major release. In recent versions,
      *   the config is optional and there is no default.
      */
+    // use FQN to avoid import deprecation warning
```

Comment:
Consider moving this line into the block above

Reply:
The above block is the javadoc, this is is an internal comment. Can you elaborate the reason for moving it?

Reply:
I have never seen javadoc followed by comment.
Since similar handling appears in other parts of the PR, maybe move the comment to description of the PR.

Reply:
The comment is just above the relevant code. I don't see how moving it to the PR is an improvement in any way.

Reply:
The visibility of PR description is higher than individual comment.

It would be good to raise awareness for other developers.

Reply:
It depends, but if you mean also adding to the PR, sure, I have done that.

Reply:
That was what I meant.